types/transaction.go

```diff
@@ -164,31 +164,30 @@ type SuggestedParams struct {
 	// Fee is in units of micro-Algos per byte.
 	// Fee may fall to zero but a group of N atomic transactions must
 	// still have a fee of at least N*MinTxnFee for the current network protocol.
-	Fee MicroAlgos
+	Fee MicroAlgos `codec:"fee"`
 
 	// Genesis ID
-	GenesisID string
+	GenesisID string `codec:"genesis-id"`
 
 	// Genesis hash
-	GenesisHash []byte
+	GenesisHash []byte `codec:"genesis-hash"`
 
 	// FirstRoundValid is the first protocol round on which the txn is valid
-	FirstRoundValid Round
+	FirstRoundValid Round `codec:"first-round"`
 
 	// LastRoundValid is the final protocol round on which the txn may be committed
-	LastRoundValid Round
+	LastRoundValid Round `codec:"last-round"`
 
 	// ConsensusVersion indicates the consensus protocol version
 	// as of LastRound.
-	ConsensusVersion string
+	ConsensusVersion string `codec:"consensus-version"`
 
-	// FlatFee indicates whether the passed fee is per-byte or per-transaction
 	// If true, txn fee may fall below the MinTxnFee for the current network protocol.
-	FlatFee bool
+	FlatFee bool `codec"flat-fee"`
```

Comment:
This is missing the  colon between tag and value

Reply:
Thanks for catching that!